app/Http/Controllers/Admin/AnswerController.php

```diff
@@ -35,7 +35,7 @@ public function store(Request $request, Question $question)
     {
         $request->validate([
             'question_id' => 'integer',
-            'answer_no'    => 'required|min:1',
+            'answer_no'    => 'required|min:1|unique:answers,answer_no',
```

Comment:
if the field is required, the value `1` for the min doesn't make sense. Please remove the min